build-system/common/exec.js

```diff
@@ -93,8 +93,10 @@ function execWithError(cmd) {
  */
 function execOrThrow(cmd) {
   const p = exec(cmd, {'stdio': ['inherit', 'inherit', 'pipe']});
-  if (p.stderr.length > 0) {
-    const error = new Error(p.stderr.toString());
+  if (p.status && p.status != 0) {
+    const error = new Error(
+      `Error executing \`${cmd}\: \n${p.stderr.toString()}`
```

Comment:
re: logging, I think the message in `build-system/tasks/cherry-pick.js` is easier to read / understand. WDYT?

Reply:
Are you suggesting we adopt the `msg` parameter and use that for the error that is thrown, or are you just referring to the `log(yellow('ERROR:'), msg)` part?

Reply:
Answered on the other comment thread.